php/Blockonomics.php

```diff
@@ -631,6 +633,10 @@ public function get_wc_order_received_url($order_id){
         return $wc_order->get_checkout_order_received_url();
     }
 
+    public function get_wc_api_url(){
+        return site_url() . "/wc-api/WC_Gateway_Blockonomics";
```

Comment:
Please refer to https://github.com/thisisayush/woocommerce-plugin/blob/435854c3589ab2abece71d097f782f343eaf2cd6/php/Blockonomics.php#L361